app/src/main/java/nerd/tuxmobil/fahrplan/congress/changes/ChangeListActivity.kt

```diff
@@ -0,0 +1,45 @@
+package nerd.tuxmobil.fahrplan.congress.changes
+
+import android.app.Activity
+import android.content.Intent
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import androidx.appcompat.widget.Toolbar
+import androidx.core.content.ContextCompat
+import nerd.tuxmobil.fahrplan.congress.R
+import nerd.tuxmobil.fahrplan.congress.base.AbstractListFragment.OnSessionListClick
+import nerd.tuxmobil.fahrplan.congress.base.BaseActivity
+import nerd.tuxmobil.fahrplan.congress.details.SessionDetailsActivity
+
+class ChangeListActivity :
+    BaseActivity(),
+    OnSessionListClick {
+
+    companion object {
+
+        @JvmStatic
+        fun start(activity: Activity) {
+            val intent = Intent(activity, ChangeListActivity::class.java)
+            activity.startActivity(intent)
+        }
+
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_change_list)
+        val toolbar = requireViewByIdCompat<Toolbar>(R.id.toolbar)
+        setSupportActionBar(toolbar)
+        val actionBarColor = ContextCompat.getColor(this, R.color.colorActionBar)
+        supportActionBar!!.setBackgroundDrawable(ColorDrawable(actionBarColor))
+        if (savedInstanceState == null) {
+            val fragment = ChangeListFragment.newInstance(sidePane = false)
+            addFragment(R.id.container, fragment, ChangeListFragment.FRAGMENT_TAG)
+        }
```

Comment:
Just wondering if this could be broke down in several pieces...looks a bit bulky to me.

Reply:
:heavy_check_mark: Addressed in 6a36eefda4857b9a76b649efcb3bdbc152db8e0b.